overlord/snapstate/aliasesv2.go

```diff
@@ -225,8 +230,8 @@ func refreshAliases(st *state.State, info *snap.Info, curAliases map[string]*Ali
 		if curTarget.Manual == "" {
 			continue
 		}
-		if info.Apps[curTarget.Manual] == nil {
-			// not an existing app
+		if !nonDaemon(info, curTarget.Manual) {
```

Comment:
I was trying not too repeat .Apps all over the place but maybe readability is more important